cmake/tinyxml2.cmake

```diff
@@ -0,0 +1,7 @@
+SET(LIB_TINYXML2_SOURCE_DIR "${LIB_SOURCE_DIR}/tinyxml2")
+SET(LIB_TINYXML2_SOURCE "${LIB_TINYXML2_SOURCE_DIR}/tinyxml2.cpp")
+SET_SOURCE_FILES_PROPERTIES(${LIB_TINYXML2_SOURCE} PROPERTIES COMPILE_FLAGS "-w")
```

Comment:
I had to look up the meaning of "-w" so maybe add a one line comment above this `# Suppress all warnings in third-party code`

Reply:
done!